packages/create-sitecore-jss/src/templates/nextjs/src/lib/next-config/plugins/cors-header.js

```diff
@@ -22,15 +22,6 @@ const corsHeaderPlugin = (nextConfig = {}) => {
             },
           ],
         },
-        {
```

Comment:
**NOTE:** CORS header for API handlers is not needed anymore, since we enforce cors in middlewares. Using Metadata architecture it's not even needed

Reply:
@art-alexeyenko I thought that the wildcards that we added should cover SITECORE_API_HOST, we will just use these values instead of SITECORE_API_HOST provided by XMCloud
@ambrauer Correct me if I'm wrong

Reply:
@illiakovalenko wildard values are used for Pages URLs only
If you want to cover CM URLs on xmcloud you'll need to use something like xmc*.cloud and adding an extra env value to allow our local deployment to continue functioning.
But that might still break SXP deployments